syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.examples.helloworld";
option java_outer_classname = "HelloWorldProto";
option objc_class_prefix = "HLW";

package test;

// The greeting service definition.
service Test {
    // Sends a greeting
    rpc TestOpen (OpenSend) returns (OpenReply) {}
  }
  
  // The request message containing the user's name.
  message OpenSend {
    string vfs_name = 1;
    string f_path = 2;
    bytes file_infor = 3;
    int32 in_flags = 4;
    int32 out_flags = 5;
  }
  
  // The response message containing the greetings
  message OpenReply {
    int32 rc = 6;
    bytes file_infor_reply = 7;
    int32 out_flags = 5;
  }
  